crates/bindings/c/src/lib.rs

```diff
@@ -1,13 +1,20 @@
 #![allow(clippy::missing_safety_doc)]
 #![allow(non_camel_case_types)]
+#[macro_use]
+extern crate lazy_static;
 
 mod types;
 
+use tokio::runtime::Runtime;
 use types::{
     blob, libsql_connection, libsql_connection_t, libsql_database, libsql_database_t, libsql_row,
     libsql_row_t, libsql_rows, libsql_rows_future, libsql_rows_future_t, libsql_rows_t,
 };
 
+lazy_static! {
+    static ref RT: Runtime = tokio::runtime::Runtime::new().unwrap();
```

Comment:
Do we want a full runtime? Or should we just create one with 1 worker thread?

Reply:
I'm happy to do one with 1 worker thread if that's enough for the core Rust library.
Just tell me if 1 thread is enough for the replicator and how to create tokio::Runtime with 1 thread :).

This is a great comment but I will push it to follow-up as I'm going for vacation today and don't want a PR hanging for a week.